app/javascript/components/miq-toolbar.jsx

```diff
@@ -45,6 +45,18 @@ const getParams = (urlParms, sendChecked) => {
   return _.filter(params).join('&') || undefined;
 };
 
+const convertMultParamsToArray = (paramName, fullParamString) => {
+  let finalString = '';
+  let index = fullParamString.search(paramName);
```

Comment:
this might become a problem if you have a param called `foo` and then another param call `foobar`, because a search for `foo` will hit on the latter.